brave/renderer/brave_content_renderer_client.cc

```diff
@@ -195,6 +197,12 @@ void BraveContentRendererClient::RenderViewCreated(
 
   new password_manager::CredentialManagerClient(render_view);
 
+  blink::WebFrameWidget* web_frame_widget = render_view->GetWebFrameWidget();
+
+  if (web_frame_widget) {
+    web_frame_widget->SetBaseBackgroundColor(SK_ColorTRANSPARENT);
+  }
```

Comment:
The Electron code checks if the `RenderView` is associated with a `webview` https://github.com/electron/electron/blob/6f2609f9d220d05588a188a4a3bbdc2c17ae90e6/atom/renderer/renderer_client_base.cc#L166-L167

@bridiver @hferreiro @darkdh Do any of you know how I can check for this in Muon?

Reply:
@samuelmaddock I think a better way to handle this would be to copy the Transparency code from WebViewGuest

Reply:
I'm still fairly new to the electron/muon codebases, do you have a link to the WebViewGuest code you had in mind?